packages/components/addon/components/hds/interactive/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{#if @route}}
+  {{#if @isRouteExternal}}
+    <LinkToExternal
```

Comment:
[question] there was some discussion over whether or not we'd need to add `ember-engines` as a dep to support this, but is my interpretation here correct that this code path will only happen if folks opt into it by passing `@isRouteExternal` meaning only apps that use engines will ever go down this path hence there isn't a big problem to solve?

Reply:
Yes. In all my tests, this works. But would be great to have feedback from the Cloud UI/TCE engineers (and more experienced Ember devs) on this, if it's an acceptable way to support external links without having to introduce engines in the design system ember setup/configuration.

Reply:
That seems logical to me and an "easy" way to solve what I thought might be pretty hard to figure out, but would be good to confirm like you said.